lib/src/main/java/com/metaplex/lib/modules/fungibletokens/models/FungibleToken.kt

```diff
@@ -1,25 +1,26 @@
 package com.metaplex.lib.modules.fungibletokens.models
 
+import com.metaplex.lib.modules.token.models.Token
 import com.metaplex.lib.programs.token_metadata.accounts.MetadataAccount
 import com.metaplex.lib.programs.token_metadata.accounts.MetaplexCollection
 import com.metaplex.lib.programs.token_metadata.accounts.MetaplexCreator
 import com.metaplex.lib.programs.token_metadata.accounts.MetaplexTokenStandard
 import com.solana.core.PublicKey
 
-class FungibleToken(
-    val metadataAccount: MetadataAccount
-) {
+typealias FungibleToken = Token
 
-    val updateAuthority: PublicKey = metadataAccount.update_authority
-    val mint: PublicKey = metadataAccount.mint
-    val name: String = metadataAccount.data.name
-    val symbol: String = metadataAccount.data.symbol
-    val uri: String = metadataAccount.data.uri
-    val sellerFeeBasisPoints: Int = metadataAccount.data.sellerFeeBasisPoints
-    val creators: Array<MetaplexCreator> = metadataAccount.data.creators
-    val primarySaleHappened: Boolean = metadataAccount.primarySaleHappened
-    val isMutable: Boolean = metadataAccount.isMutable
-    val editionNonce: Int? = metadataAccount.editionNonce
-    val tokenStandard: MetaplexTokenStandard? = metadataAccount.tokenStandard
-    val collection: MetaplexCollection? = metadataAccount.collection
-}
\ No newline at end of file
+//class FungibleToken(metadataAccount: MetadataAccount) : Token(metadataAccount) {
+//
+//    val updateAuthority: PublicKey = metadataAccount.update_authority
+//    val mint: PublicKey = metadataAccount.mint
+//    val name: String = metadataAccount.data.name
+//    val symbol: String = metadataAccount.data.symbol
+//    val uri: String = metadataAccount.data.uri
+//    val sellerFeeBasisPoints: Int = metadataAccount.data.sellerFeeBasisPoints
+//    val creators: Array<MetaplexCreator> = metadataAccount.data.creators
```

Comment:
oh good catch

I was also thinking I should maybe pull the common code from NFTClient and FugibleTokenClient into a shared abstract TokenClient since they are basically the same for findByMint. What do you think?